tools/roadmap/validate_dag.py

```
@@ -0,0 +1,68 @@
+    # Basic syntax lint via mermaid-cli if available (optional)
+    try:
+        run(["docker", "run", "--rm", "-v", f"{MMD.parent.resolve()}:/data", "minlag/mermaid-cli", "-i", "/data/ROADMAP-DAG.mmd", "-o", "/data/.lint.svg"])
+    except Exception as e:
+        print(f"validate_dag: warning: mermaid lint skipped: {e}")
+
+    # Sanity: all issue numbers exist
+    missing = []
+    for num in node_to_issue.values():
+        try:
+            gh_json(["issue", "view", str(num), "--json", "number" ])
+        except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)